# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-flexbox/inline-flexbox-vertical-rl-image-flexitem-crash-print.html
# Commit: 42ed600d09bb
# Full Hash: 42ed600d09bbe21c0f30b07d68591758f052e314
# Author: Ting-Yu Lin <tlin@mozilla.com>
# Date: 2022-12-15 09:27:59
# Description:
#   Bug 1805522 - Construct SharedFlexData for incomplete flex containers, regardless of whether they have constrained BSize. r=dholbert
#   
#   This reverts part of Bug 1803486 Part 1.
#   
#   The crashtest was adapted from bug 1805522 comment 0. I use wpt printing test
# ==============================================================================

diff -r b581a525e26e -r 42ed600d09bb testing/web-platform/tests/css/css-flexbox/inline-flexbox-vertical-rl-image-flexitem-crash-print.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-flexbox/inline-flexbox-vertical-rl-image-flexitem-crash-print.html	Wed Dec 14 20:51:59 2022 +0000
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="author" title="Ting-Yu Lin" href="mailto:tlin@mozilla.com">
+<link rel="author" title="Mozilla" href="https://www.mozilla.org/">
+<link rel="help" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1805522">
+<!-- This is a printing crashtest, but wpt doesn't support it.
+     We work around it by ensuring the test renders something non-blank. -->
+<link rel="mismatch" href="/common/blank.html">
+<style>
+#a {
+  height: 0.473em;
+  writing-mode: vertical-rl;
+  display: -webkit-inline-box;
+}
+#b {
+  overflow-x: scroll;
+  height: fit-content;
+  content: url(data:image/gif;base64,R0lGODlhEAAQAMQAAORHHOVSKudfOulrSOp3WOyDZu6QdvCchPGolfO0o/XBs/fNwfjZ0frl3/zy7////wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAkAABAALAAAAAAQABAAAAVVICSOZGlCQAosJ6mu7fiyZeKqNKToQGDsM8hBADgUXoGAiqhSvp5QAnQKGIgUhwFUYLCVDFCrKUE1lBavAViFIDlTImbKC5Gm2hB0SlBCBMQiB0UjIQA7)
+}
+</style>
+<dl id="a">
+<dd id="b">
